Vostok.Logging.File/Configuration/RollingStrategyOptions.cs

```diff
@@ -27,5 +27,10 @@ public class RollingStrategyOptions
         /// Maximal size of one part of log file, in bytes. Affects only <see cref="RollingStrategyType.BySize"/> and <see cref="RollingStrategyType.Hybrid"/> strategies.
         /// </summary>
         public long MaxSize { get; set; } = 1024 * 1024 * 1024;
+
+        /// <summary>
+        /// Eliminator between base path and rolling suffix. Not affects <see cref="RollingStrategyType.None"/> strategy.
+        /// </summary>
+        public char Eliminator { get; set; } = '-';
```

Comment:
Why is it called an eliminator? What exactly does it eliminate? :)
Something like 'SuffixSeparator' might be more appropriate.